scripts/pdf/render.py

```diff
@@ -67,14 +76,23 @@ def main(loc, colorscheme, output_filename):
             html += '<p style="page-break-before: always" ></p>'
             print(f"Rendered page {page_number} of the directory {operating_sys}")
 
+    output_filename_with_platform = output_filename
```

Comment:
This feels weird, where if `platform = None`, then the `with_platform` bit here is wrong as there's no `platform` to add.

I'd just rewrite `output_filename` as the parameter itself isn't used after you have `output_filename_with_platform`.